### ISSUE TITLE
Access app with my local network ip

### ISSUE THREAD

**Issue by rccc** [dunglas/symfony-docker#603]
Hello,

Il i try to connect with https:localhost, it works, but not with my own local ip !

How can i make it works with, for example 192.1.168.xxx or 192.1.168.xxx:8086.

Many thanks in advance.

**Comment by maxhelias** [comment]
Hello,

It's a firewall or network issue, check your different configurations.

**Comment by 7-zete-7** [comment]
Hello, @rccc!

You don't need to use your machine's local address for FrankenPHP. FrankenPHP (it's Docker container) does not have this address at his disposal and cannot listen to ports on this address. Use `SERVER_NAME=:80` to access the container from outside of your machine.

Reference to Caddy's addresses rules: https://caddyserver.com/docs/caddyfile/concepts#addresses

**Comment by rccc** [comment]
Hello @7-zete-7,

Many thanks for your kind response !

The link is a very interesting resources.

But if SERVER_NAME is "my_app", i do not think that https:://my_app:80 will be reachable from my local network unless  i edit the hosts file on each computer in my local network... But i may wrong ...

Regards

**Comment by 7-zete-7** [comment]
@rccc, yes, without editing `/etc/hosts` file, the site will not open using `https://my_app:80` link. Moreover, FrankenPHP will wait at port `80` only for HTTP, not HTTPS.

As a rule, they use the `localhost` host to simplify the deployment of a container for development.

If you need to use a specific host, you must add it to the `/etc/hosts` file manually by specifying the IP address `127.0.0.1` (or the external address of your host).

#### Examples

_/etc/hosts_
```
...
my_host 127.0.0.1
...
```

Starting a container
```shell
SERVER_NAME=my_app ...
```
